Formula/fluent-bit.rb

```diff
@@ -26,13 +26,15 @@ class FluentBit < Formula
   end
 
   def install
-    # Per https://luajit.org/install.html: If MACOSX_DEPLOYMENT_TARGET
-    # is not set then it's forced to 10.4, which breaks compile on Mojave.
-    # fluent-bit builds against a vendored Luajit.
-    ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
+    chdir "build" do
+      # Per https://luajit.org/install.html: If MACOSX_DEPLOYMENT_TARGET
+      # is not set then it's forced to 10.4, which breaks compile on Mojave.
+      # fluent-bit builds against a vendored Luajit.
+      ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
 
-    system "cmake", ".", "-DWITH_IN_MEM=OFF", *std_cmake_args
-    system "make", "install"
+      system "cmake", "..", "-DWITH_IN_MEM=OFF", "-DCMAKE_INSTALL_PREFIX:PATH=#{prefix}", *std_cmake_args
```

Comment:
```suggestion
      system "cmake", "..", "-DWITH_IN_MEM=OFF", *std_cmake_args
```
`CMAKE_INSTALL_PREFIX` is already in `std_cmake_args`.

Reply:
ok I'll remove that thank you

Reply:
Fixed thank you